tests/simple/simple_test.go

```diff
@@ -113,30 +130,95 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-func TestSimpleFile(t *testing.T) {
-	// Special case to handle test invocation without args. See TestMain()
-	// early return comment.
-	if rc == nil {
-		return
-	}
-	// Run the flag-configured tests.
-	for _, filename := range flag.Args() {
-		testFile, err := parseSimpleFile(filename)
-		if err != nil {
-			t.Fatalf("Can't parse input file %v: %v", filename, err)
+ func TestSimpleFile(t *testing.T) {
+        // Special case to handle test invocation without args. See TestMain()
+        // early return comment.
+        if rc == nil {
+                return
+        }
+	var skipTests []string
+	firstFlag := true
+	for _, flagVal := range skipFlags {
+		if !strings.Contains(flagVal, "/") {
+			log.Fatal("Unable to parse skip_test flag, specifically for ", flagVal)
+		}
+		fileInd := strings.Index(flagVal, "/")
+		if (flagVal[:fileInd] == ""){
```

Comment:
Run it through "gofmt" so it can catch things like `){` which should be `) {`.

If there is no "/" in the flag value, then Index() returns -1, and I _think_ the slice expression gives a runtime panic.  Check instead that fileInd > 0, which rules out both a missing "/" and a leading "/".